roles/splunk/tasks/kvstore_upgrade.yml

```diff
@@ -0,0 +1,156 @@
+---
+- name: Check if authenticated
+  include_tasks: splunk_login.yml
+  when: not splunk_authenticated
+
+- name: Set fact for kvstore Upgrade
+  ansible.builtin.set_fact:
+    splunk_upgrade_server: false
```

Comment:
Where is this var being used?

Reply:
Oof, that's a headpalm

Reply:
Fixed in https://github.com/splunk/ansible-role-for-splunk/commit/3263c35509de7ece33be496ed9f6b0fbead62f94